ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -0,0 +1,159 @@
+4074.2 "Merciful Arc" sync /:Trinity Seeker:5AD4:/
+
+# Initial Baleful Air (80% or time push)
+4076.5 "--sync--" sync /:5A98:Trinity Seeker/ window 80,20
+4079.5 "Verdant Path" sync /:Trinity Seeker:5A98:/
+4084.1 "Baleful Swathe" sync /:Trinity Seeker:5AD0:/
+4092.2 "Baleful Onslaught" sync /:Trinity Seeker:5AD5:/
+4099.3 "Phantom Edge?" sync /:Trinity Seeker:5ABD:/
+4105.5 "Baleful Onslaught" sync /:Trinity Seeker:5AD6:/
+4117.8 "--sync--" sync /:Trinity Seeker:5A9A:/
+4121.7 "Scorching Shackle" sync /:Trinity Seeker:5ACB:/
+4126.9 "Baleful Blade" sync /:Trinity Seeker:5ABE:/
+4132.1 "Manifest Avatar" sync /:Trinity Seeker:5ADA:/
+4138.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4139.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4140.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4141.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4142.2 "Phantom Edge" sync /:Trinity Seeker:5ABD:/
+4144.4 "--sync--" sync /:Trinity Seeker:5A9A:/
+4146.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4148.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4150.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4152.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4153.5 "Baleful Blade" sync /:Trinity Seeker:5ABF:/
+4162.6 "Verdant Tempest" sync /:Trinity Seeker:5AD3:/
+4171.7 "Verdant Path" sync /:Trinity Seeker:5A99:/
+
+# TODO: in progress
+
+### Stygimoloch Warrior
+# TODO: this and Dahu can be seen from each other.
+# Jump here via injected "1961" auto? or some other ability similar to castrum timeline.
+#6000.0 "--sync--" sync / 00:0839:Pride of the Lioness will be sealed off/ window 10000,0
+#6000.0 "--Reset--" sync / 00:0839:Pride of the Lioness is no longer sealed/ window 0,2000 jump 0
+
+### Dahu
+# -p 576D:8008.4
+# -ii 576C 5B43
+8000.0 "--sync--" sync / 00:0839:The Hall of Supplication will be sealed off/ window 10000,0
+8000.0 "--Reset--" sync / 00:0839:The Hall of Supplication is no longer sealed/ window 0,2000 jump 0
+8008.4 "Reverberating Roar" sync /:Dahu:576D:/ window 10,2.5
+8013.5 "Falling Rock x6" duration 10 #sync /:Dahu:576E:/
+8014.5 "Reverberating Roar" sync /:Dahu:576D:/
+8022.7 "Hot Charge 1" sync /:Dahu:5773:/
+8027.5 "Hot Charge 2" sync /:Dahu:5773:/
+8034.3 "Firebreathe" sync /:Dahu:5774:/
+8046.5 "--sync--" sync /:Marchosias:576A:/
+8051.8 "Head Down" sync /:Marchosias:5768:/
+8057.5 "Head Down" sync /:Marchosias:5768:/
+8061.4 "Spit Flame" sync /:Dahu:5776:/
+8061.6 "Spit Flame 1" #sync /:Dahu:5777:/
+8062.7 "Spit Flame 2" #sync /:Dahu:5777:/
+8063.8 "Spit Flame 3" #sync /:Dahu:5777:/
+8065.0 "Spit Flame 4" #sync /:Dahu:5777:/
+8069.1 "Head Down" sync /:Marchosias:5768:/
+8071.1 "Right-Sided Shockwave/Left-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8073.7 "Left-Sided Shockwave/Right-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8074.8 "Head Down" sync /:Marchosias:5768:/
+8081.4 "Feral Howl" sync /:Dahu:5767:/
+8083.5 "--knockback--" sync /:Dahu:5B35:/ # TODO: confirm
```

Comment:
5B35 is the knockback and damage. 5767 is the cast on the boss, but not what has the knockback and damage.